src/Shared/Resources/Strings.shared.resx

```diff
@@ -262,10 +262,22 @@
     <value>Path: {0} exceeds the OS max path limit. The fully qualified file name must be less than {1} characters.</value>
     <comment></comment>
   </data>
+  <data name="SolutionFilterJsonParsingError" Visibility="Public">
+    <value>MSB5025: Json in solution filter file "{0}" is incorrectly formatted.</value>
+    <comment>{StrBegin="MSB5025: "}UE: The solution filename is provided separately to loggers.</comment>
+  </data>
+  <data name="SolutionFilterMissingSolutionError" Visibility="Public">
+    <value>MSB5026: The solution filter file at "{0}" specifies there will be a solution file at "{1}", but that file does not exist.</value>
+    <comment>{StrBegin="MSB5026: "}UE: The solution filename is provided separately to loggers.</comment>
+  </data>
+  <data name="SolutionFilterFilterContainsProjectNotInSolution" Visibility="Public">
```

Comment:
Will this result in duplicated errors for a missing project file?
I see that when building a solution we get 

> MSB3202: The project file "<>" was not found.

 for a project in the solution that does not exist.

Reply:
MSB3202 is for if a project in the .sln doesn't exist on disk; this new error is for if a project is in the filter but is not in the solution file. Together, they ensure that any project in the .slnf is in both the .sln and on disk as desired, but they can never both be thrown because one requires the project to be in the .sln and the other cannot have the project in the .sln.

Reply:
Ah, got it. Thanks for the explanation.